### DISCUSSION TITLE
Private package with yarn v2

### DISCUSSION THREAD

**Discussion by cklok1997** [renovatebot/renovate#11052] [Request Help]
Currently my package having 2 dependencies, may i know is renovate support update package by using gitlab repo url? If yes can provide example of configuration needed based on the dependencies I given?

"dependencies": {
    "lodash": "^4.17.21",
    "package_name": "http://gitlab.comapny.com/owner/project.git#semver=^1.0.0"
  },

**Comment by rarkins** [comment]
I'm not sure this is currently supported. Please run with `LOG_LEVEL=debug` in env and then look at the log message `packageFiles with updates` to see what is said about this dependency.

**Comment by cklok1997** [comment]
{
  "depType": "dependencies",
  "depName": "package_name",
  "currentValue": "http://gitlab.comapny.com/owner/project.git#semver=^1.0.0",
  "skipReason": "unknown-version",
  "prettyDepType": "dependency",
  "depIndex": 1,
  "updates": []
}

I think it cant identify url format, this case fall to github condition.
if (githubRepoSplit.length !== 2) {
      // stop here
     dep.skipReason = types_1.SkipReason.UnknownVersion;
     return dep;
}

**Reply by rarkins** [reply]
Yes, that confirms that we'd need an enhancement to our npm extract logic to support this. Please raise a feature request with minimal reproduction repo if you'd like
